wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,332 @@
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | June 2009 | July 2009 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | July 2009 | 2009-07-05 |  |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2009-07-05 | July 2009 | [\[1\]](https://web.archive.org/web/20090714162313/https://osu.ppy.sh/u/18983 "Internet Archive — osu!") [\[2\]](https://www.youtube.com/watch?t=190&v=0T5F0Zzy6ps "YouTube") [\[3\]](https://osu.ppy.sh/scores/osu/25301195) |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | July 2009 | 2009-08-13 |  |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2009-08-13 | 2009-09-02 | [\[1\]](https://osu.ppy.sh/community/forums/topics/14784?start=179293) |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | 2009-09-02 | October 2009 | [\[1\]](https://web.archive.org/web/20090903052009/http://osu.ppy.sh/p/playerranking/?m=0&s=2&r=&q=&f= "Internet Archive — osu!") [\[2\]](https://osu.ppy.sh/community/forums/topics/17003?start=194808) [\[3\]](https://www.youtube.com/watch?t=200&v=0T5F0Zzy6ps "YouTube") |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | October 2009 | November 2009 |  |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | November 2009 | 2009-12-08 |  |
+| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | 2009-12-08 | *2010-02-07* | [\[1\]](https://osu.ppy.sh/community/forums/topics/20010?start=262589) |
+
+## 2010
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | *2009-12-08* | 2010-02-07 | [\[1\]](https://osu.ppy.sh/community/forums/topics/20010?start=262589) |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | 2010-02-07 | 2010-07-10 | [\[1\]](https://web.archive.org/web/20100207194753/http://osu.ppy.sh/p/playerranking "Internet Archive — osu!") |
+| ![][flag_KR] [Sure](https://osu.ppy.sh/users/70730) | 2010-07-10 | 2010-08-10 | [\[1\]](https://imgur.com/a/cF4mZqm "Imgur") |
+| ![][flag_JP] [zaru-d](https://osu.ppy.sh/users/79678) | 2010-08-10 | September 2010 | [\[1\]](https://osu.ppy.sh/community/forums/topics/35178) [\[2\]](https://osu.ppy.sh/community/forums/topics/35345) |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | September 2010 | October 2010 |  |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | October 2010 | November 2010 |  |
+| ![][flag_JP] [zaru-d](https://osu.ppy.sh/users/79678) | November 2010 | November 2010 |  |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | November 2010 | 2010-11-28 |  |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2010-11-28 | *2011-03-20* | [\[1\]](https://imgur.com/VlP4LOb "Imgur") [\[2\]](https://www.youtube.com/watch?t=555&v=0T5F0Zzy6ps "YouTube") [\[3\]](https://www.youtube.com/watch?t=785&v=0T5F0Zzy6ps "YouTube") |
```

Comment:
Updating 2010 to match our current sources:

```suggestion
| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | *2009-12-08* | January 2010 | [\[1\]](https://osu.ppy.sh/community/forums/topics/20010?start=262589) |
| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | January 2010 | 2010-02-01 | [\[1\]](https://web.archive.org/web/20100207194753/http://osu.ppy.sh/p/playerranking "Internet Archive — osu!") [\[2\]](https://osu.ppy.sh/community/forums/topics/23308?start=315697) |
| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | 2010-02-01 | 2010-02-05 | [\[1\]](https://osu.ppy.sh/community/forums/topics/23715) |
| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | 2010-02-05 | 2010-03-17 | [\[1\]](https://web.archive.org/web/20100207194753/http://osu.ppy.sh/p/playerranking "Internet Archive — osu!") [\[2\]](https://osu.ppy.sh/community/forums/topics/23308?start=315697) |
| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2010-03-17 | 2010-06-07 | [\[1\]](https://osu.ppy.sh/community/forums/topics/26603)  |
| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | 2010-06-07 | 2010-06-25 | [\[1\]](https://osu.ppy.sh/community/forums/topics/30241) |
| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | 2010-06-25 | 2010-07-10 | [\[1\]](https://osu.ppy.sh/community/forums/topics/30241?start=416333) |
| ![][flag_KR] [Sure](https://osu.ppy.sh/users/70730) | 2010-07-10 | 2010-08-10 | [\[1\]](https://imgur.com/a/cF4mZqm "Imgur") |
| ![][flag_JP] [zaru-d](https://osu.ppy.sh/users/79678) | 2010-08-10 | September 2010 | [\[1\]](https://osu.ppy.sh/community/forums/topics/35178) [\[2\]](https://osu.ppy.sh/community/forums/topics/35345) |
| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | September 2010 | October 2010 |  |
| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | October 2010 | November 2010 |  |
| ![][flag_JP] [zaru-d](https://osu.ppy.sh/users/79678) | November 2010 | November 2010 |  |
| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | November 2010 | 2010-11-28 |  |
| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2010-11-22 | *2011-03-20* | [\[1\]](https://imgur.com/VlP4LOb "Imgur") [\[2\]](https://www.youtube.com/watch?t=555&v=0T5F0Zzy6ps "YouTube") [\[3\]](https://www.youtube.com/watch?t=785&v=0T5F0Zzy6ps "YouTube") |
```

It feels like we are still missing a bunch of changes during the first portion of the year... oh well...

Reply:
Resolving since no longer applicable to the current document as it has been added already.